safekeeper/src/timelines_global_map.rs

```diff
@@ -66,6 +72,7 @@ static TIMELINES_STATE: Lazy<Mutex<GlobalTimelinesState>> = Lazy::new(|| {
         conf: None,
         broker_active_set: Arc::new(TimelinesSet::default()),
         load_lock: Arc::new(tokio::sync::Mutex::new(TimelineLoadLock)),
+        partial_backup_rate_limiter: RateLimiter::new(1),
```

Comment:
Why is this hard-coded to 1?

Reply:
Because it's "empty" value inside a static global variable. The value here doesn't matter, it will get overriden in `GlobalTimelines::init()`.

Can be done better, but it's better to fix https://github.com/neondatabase/neon/issues/8200 instead.